ui/components/extensions/adapters/constants.js

```diff
@@ -0,0 +1,115 @@
+const adaptersDescription = (adapterName) => {
+  return `Deploy the Meshery Adapter for ${adapterName} in order to enable deeper lifecycle management of ${adapterName}.`;
+}
+
+export const adaptersList = {
+  "ISTIO" : {
+    name : "Istio",
+    label : "meshery-istio",
+    imageSrc : "/static/img/istio.svg",
+    description : adaptersDescription("Istio"),
+    defaultPort : 10000,
+    enabled : false,
+    url : "",
+  },
+  "LINKERD" : {
+    name : "Linkerd",
+    label : "meshery-linkerd",
+    imageSrc : "/static/img/linkerd.svg",
+    description : adaptersDescription("Linkerd"),
+    defaultPort : 10001,
+    enabled : false,
+    url : "",
+  },
+  "CONSUL" : {
+    name : "Consul",
+    label : "meshery-consul",
+    imageSrc : "/static/img/consul.svg",
+    description : adaptersDescription("Consul"),
+    defaultPort : 10002,
+    enabled : false,
+    url : "",
+  },
+  "meshery-octarine" : {
+    name : "Meshery Octarine",
+    label : "meshery-octarine",
```

Comment:
Difference between name and label?
Which is user facing?

Reply:
Name is basically the Name of the adapter on the card. While label is the used for the deployment api payload.

Reply:
Please add this comment on code.